### ISSUE TITLE
Fyne sometimes renders UI incorrectly offset vertically 

### ISSUE THREAD

**Issue by Bradbev** [fyne-io/fyne#1259]
### Describe the bug:
The attached code when run on my machine will sometimes (roughly 1 out of 10) render buttons offset vertically.

### To Reproduce:
This is a minimal test case
```package main

import (
	"fyne.io/fyne"
	"fyne.io/fyne/app"
	"fyne.io/fyne/widget"
)

func main() {
	a := app.New()
	w := a.NewWindow("Test Runner")
	w.Resize(fyne.NewSize(800, 800))

	controlSection := widget.NewVBox(
		widget.NewButton("Start", func() {}),
		widget.NewButton("Cancel", func() {}),
	)
	w.SetContent(controlSection)

	w.ShowAndRun()
}
```

### Screenshots:
Sometimes looks like 
![image](https://user-images.githubusercontent.com/36225/90968412-1454f080-e540-11ea-9ab7-d52f8536351a.png)
Should always look like 
![image](https://user-images.githubusercontent.com/36225/90968420-259dfd00-e540-11ea-8b34-e747eebb3e03.png)

### Device
Fyne 1.3.3
go version 1.15
```
✔ ~/development/gomod/fyne-bug 12:53 $ uname -a
Darwin macbook-pro-7.lan 18.6.0 Darwin Kernel Version 18.6.0: Thu Apr 25 23:16:27 PDT 2019; root:xnu-4903.261.4~2/RELEASE_X86_64 x86_64
✔ ~/development/gomod/fyne-bug 12:53 $ sw_vers -productVersion
10.14.5```

**Comment by Bradbev** [comment]
Sorry - go 1.15 darwin/amd.  Fyne 1.3.3

**Comment by andydotxyz** [comment]
Can anyone with macOS 10.14 replicate this? I cannot seem to make it happen. @toaster maybe?

**Comment by andydotxyz** [comment]
Also, is this still an issue with `v1.4.0` @Bradbev

**Comment by Bradbev** [comment]
@andydotxyz - I'm afraid I can still reproduce it.  No code changes, just an update to v1.4.0.  Sorry!

**Comment by andydotxyz** [comment]
I'm sorry to hear this @Bradbev.
As `widget.Box` is now deprecated, can you try replacing it with `container.NewVBox` and see if it helps.
When you are able to capture the failure can you send an updated screenshot please?
Thanks

**Comment by Bradbev** [comment]
Same with `container.NewVBox` I'm afraid.
![image](https://user-images.githubusercontent.com/36225/100068068-e57e4b80-2e9b-11eb-9be5-5595c542b385.png)

**Comment by toaster** [comment]
> Can anyone with macOS 10.14 replicate this? I cannot seem to make it happen. @toaster maybe?

Sorry, I don’t longer have 10.14. Was forced to the crappy 10.15 due to hardware upgrade (at least the new keyboards are no longer crap).

**Comment by RustamGamidov** [comment]
reproduced with one change to the code:
  the window height is set to the value bigger than the screen height (800x1200) in my case.
env:
  - macOS 10.15.7
  - go 1.15.5
  - fine 57a94898

**Comment by andydotxyz** [comment]
Oh this is really interesting thanks @RustamGamidov.
Is it possible @Bradbev that your window was the height of the screen as well (I can't tell for sure from your screenshots)?

**Comment by Bradbev** [comment]
Sorry for the delay.  Yes!  I'm pretty sure this is the issue.  I had set my window to 800x800.   It looks like the actual window gets resized to the max height of my screen.  If I set the height to be less than my screen (about 750) then everything is fine.  Conversely, if I set height to be greater than 800, then the offset of the buttons moves further down.
Resizing with mouse or fullscreen fixes the render issue.  Moving the window does not fix it.
Great work @RustamGamidov !

**Comment by Jacalz** [comment]
Is this still and issue for people, using the latest release? I have never personally seen this.

**Comment by dweymouth** [comment]
I don't think this is an issue anymore. If it still is for anyone, feel free to comment to reopen.
